Configuration/Generator/python/VBFHZZ4Nu_14TeV_cfi.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         #pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(14000.0),
+                         maxEventsToPrint = cms.untracked.int32(0),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         crossSection = cms.untracked.double(0.004783), # 4.233 pb * 0.00113 (qqH * BR(4v))
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        processParameters = cms.vstring(
+            'HiggsSM:ff2Hff(t:ZZ) = on',
+            'HiggsSM:ff2Hff(t:WW) = on',
```

Comment:
Do you want to add a fixed mass to keep it stable against changes of P8 default?

Reply:
Good idea. Updated to 125.